_rules/inline-link-in-paragraph-has-distinguishable-style-not-based-on-color-alone-88407d.md

```diff
@@ -27,16 +27,17 @@ This rule applies to any [visible][] HTML element that is a [semantic link][], f
 
 For each test target, at least one of the following is true:
 
-- **distinguishing style**: the element has a [distinguishing style][] not based on color alone from the **non-link line text** elements; or
-- **distinguishing content**: element has content (such as an image or text), located in the proximity of the element (i.e. there is no other content visually in between this content and the target element), that indicates the test target is a link.
+- **distinguishing style**: the element has a [distinguishable style][] not based on color alone from the **non-link line text** elements; or
+- **distinguishing content**: the element has content (such as an image or text), located in the proximity of the element (i.e. there is no other content visually in between this content and the target element), that indicates the test target is a link.
 
 ## Assumptions
 
-- The 3:1 contrast difference between text is minimal to what would be sufficient to meet WCAG 2.0. This value is part of [technique G183](https://www.w3.org/WAI/WCAG21/Techniques/general/G183) but is not specified in the [1.4.1 success criterion](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html).
+- The 3:1 contrast difference between text color is the minimum required to meet WCAG 2.0. This value is part of [technique G183](https://www.w3.org/WAI/WCAG21/Techniques/general/G183) but is not specified in the [1.4.1 success criterion](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html).
```

Comment:
I still feel that doesn't really read as an assumption.
As far as I can tell, the assumption is that "something with a contrast lower than 3:1 is not concerned by 1.4.1".

This does tie a bit with https://github.com/act-rules/act-rules.github.io/pull/1010#discussion_r459320027 If the link does not have that contrast ratio, which SC is it failing (if any)? I feel it should definitely fail something, but I can't find what…

Reply:
Rephrased it. I think it's easier to understand now.